modules/claims_api/app/swagger/claims_api/forms/veteran_identifier_example.json

```diff
@@ -0,0 +1,8 @@
+{
+  "attributes": {
+    "veteranSSN": "796130115",
+    "veteranFirstName": "Tamara",
+    "veteranLastName": "Ellis",
+    "veteranBirthDate": "1967-06-19"
```

Comment:
can we just do:
- ssn
- firstName
- lastName
- birthdate

for these?

Reply:
Changed [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6845/commits/a3aae693d14ff06137846080cac3634f9c2a8c5d).